app/server/models/reactive_actuator.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+from typing import Optional, Union
+
+from bson import ObjectId
+from dotenv import load_dotenv
+from pydantic import BaseModel, Field, root_validator
+
+load_dotenv()
+
+
+from .id import PyObjectId
+
+
+class ReactiveActuatorModel(BaseModel):
+    id: PyObjectId = Field(default_factory=PyObjectId, alias="_id")
+    name: str = Field(...)
+    sensor_id: Union[PyObjectId, None] = Field(default=None, alias="sensor_id")
```

Comment:
I think this is allowing a reactive actuator to be put in without a sensor_id, we don't want to let that happen

Reply:
Also, I guess the autoformatter / linter didn't work (imports are out of order and spaced out weird), remind me or Jonas to help figure out how to get that working for you